app/models/service.rb

```diff
@@ -97,11 +97,11 @@ def direct_vms
   end
 
   def all_vms
-    MiqPreloader.preload_and_map(subtree, :direct_vms)
+    MiqPreloader.preload_and_map(subtree, :vms)
```

Comment:
@bzwei @gmcculloug I probably did this wrong, it should just call all_vms, but since subtree contains itself it can't

Btw. what was the reason of all_vms to call just direct vms? We need all_vms to show vms in nested stacks